meson_options.txt

```diff
@@ -8,7 +8,8 @@ option('enable-uefi', type : 'boolean', value : true, description : 'enable UEFI
 option('enable-uefi-labels', type : 'boolean', value : true, description : 'enable UEFI labels support')
 option('enable-dell', type : 'boolean', value : true, description : 'enable Dell-specific support')
 option('enable-amt', type : 'boolean', value : true, description : 'enable Intel AMT support')
-option('enable-tbtfwu', type : 'boolean', value : true, description : 'enable Thunderbolt support')
+option('enable-tbtfwu', type : 'boolean', value : true, description : 'enable legacy Thunderbolt support')
```

Comment:
Option - sure, the question is about the default. But this a policy question that depends on the approach you decided for the project, so this was just a suggestion.

Just a reminder, Intel announced the integration of TBT controler with the CPU and is working towards opening the spec to the public usage, so the expectation is to see many more TBT controllers in the field.